specification/resources/resource-manager/Microsoft.Resources/regionalmove/preview/2026-01-01-preview/regionalMove.json

```diff
@@ -0,0 +1,700 @@
+      "properties": {
+        "tags": {
+          "type": "object",
+          "description": "Resource tags.",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "RegionalMoveFinalizationRequestContent": {
+      "type": "object",
+      "description": "Represents the finalization request body.",
+      "properties": {
+        "resourceIds": {
+          "type": "array",
+          "description": "List of fully-qualified resource IDs to mark for finalization. Each resource ID specified should be part of the Regional Move Container.\nIf you set this to null, all resources in the move container will be marked for finalization.",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "RegionalMoveProvisioningState": {
+      "type": "string",
+      "description": "The regional move container provisioning state",
+      "enum": [
+        "Succeeded",
+        "Failed",
+        "Canceled",
+        "Created",
+        "InProgress",
+        "InProgressAwaitingFinalization"
+      ],
+      "x-ms-enum": {
+        "name": "RegionalMoveProvisioningState",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "Succeeded",
+            "value": "Succeeded",
+            "description": "Resource has been created."
+          },
+          {
+            "name": "Failed",
+            "value": "Failed",
+            "description": "Resource creation failed."
+          },
+          {
+            "name": "Canceled",
+            "value": "Canceled",
+            "description": "Resource creation was canceled."
+          },
+          {
+            "name": "Created",
+            "value": "Created",
+            "description": "The regional move container has been successfully created but not started."
+          },
+          {
+            "name": "InProgress",
+            "value": "InProgress",
+            "description": "The regional move container operations are in progress."
+          },
+          {
+            "name": "InProgressAwaitingFinalization",
+            "value": "InProgressAwaitingFinalization",
+            "description": "The regional move container is awaiting the user to invoke finalization on one or more resources."
+          }
+        ]
+      },
+      "readOnly": true
+    },
+    "RegionalMoveRequestedResource": {
+      "type": "object",
+      "description": "Information about the resource to be copied",
+      "properties": {
+        "sourceResourceId": {
+          "type": "string",
+          "description": "Resource ID of the resource being copied"
+        },
+        "targetResourceId": {
+          "type": "string",
+          "description": "Resource ID where the source will be copied/moved to. For copy operations, this cannot be the same as the source."
+        },
+        "resourceProperties": {
```

Comment:
>resourceProperties [](http://example.com/codeflow?start=9&length=18)

Could you update the examples to show this property too ?

Reply:
Will do.